easybuild/easyconfigs/n/NextClip/NextClip_scripts_shebang.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/scripts/index_bwa.sh b/scripts/index_bwa.sh
+index 2862e42..472d7f1 100755
+--- a/scripts/index_bwa.sh
++++ b/scripts/index_bwa.sh
+@@ -1,4 +1,4 @@
+-#!/bin/bash
++#!/usr/bin/env bash
```

Comment:
well, ok, but we don't build our own bash?

I would really prefer keeping `/bin/bash` here (and below), I don't think consistency should overrule common practice (with the exception of `/usr/bin/perl` being wrong for technical reasons)